packages/@aws-cdk/aws-events/README.md

```diff
@@ -114,6 +114,7 @@ The following targets are supported:
 * `targets.SqsQueue`: Send a message to an Amazon SQS Queue
 * `targets.SfnStateMachine`: Trigger an AWS Step Functions state machine
 * `targets.AwsApi`: Make an AWS API call
+* `targets.BatchJob`: COMING SOON
```

Comment:
At AWS we make a habit of not promising things so that we don't have to disappoint our customers if for whatever reason we are unable to deliver.  We will advertise this new class when it's finished.

Reply:
Thank you for the insight. I’m on the same page.

Reply:
removed